docker-compose.windows.yml

```diff
@@ -0,0 +1,24 @@
+version: '2'
+services:
+  db:
+    image: microsoft/mssql-server-2016-express-windows
+    environment:
+      sa_password: "Password1"
+    ports:
+      - "1433:1433" # REMARK: This is currently required, needs investigation
+
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile.windows
+    environment:
+      - "Data:DefaultConnection:ConnectionString=Server=db,1433;Database=MusicStore;User Id=sa;Password=Password1;MultipleActiveResultSets=True"
```

Comment:
Comma is the canonical representation, and just want to echo this commenter's sentiment: http://stackoverflow.com/questions/5294721/how-to-specify-a-port-number-in-sql-server-connection-string#comment27379349_5294741